tox.ini

```diff
@@ -53,6 +53,10 @@ commands =
 envdir = .tox/lint
 commands = black {posargs} qiskit test tools examples setup.py
 
+[testenv:ruff]
+envdir = .tox/lint
+commands = ruff {posargs} check qiskit test tools examples setup.py
+
```

Comment:
I think we probably don't need a separate environment if it's replacing `pylint` - it's just one more thing to get out-of-sync.

Reply:
Removed in 7a2889eeafbf4d824fea345093f73918cc4cc4ac